bin.js

```diff
@@ -74,17 +74,32 @@ const packageDetails = JSON.parse(
       throw "Didn't find Y(A)ML files in provided paths: " + program.args;
     }
 
+    const shouldFailFast = filesToProcess.length === 1;
+
     for (const filename of filesToProcess) {
       mainDebug("Processing " + filename);
       const input = fs.readFileSync(filename).toString();
-      const output = await run(
-        input,
-        allowed,
-        ignoreShas,
-        allowEmpty,
-        debug,
-        comment,
-      );
+      let output;
+      try {
+        output = await run(
```

Comment:
Obviously could instead add a failFast parameter that flows down to all these methods but I think try catch is sufficient without adding downstream complexity.